### DISCUSSION TITLE
Since I updated to next 13, i cant deploy to vercel due the serverless functions maximum size limit

### DISCUSSION THREAD

**Discussion by gmotta-dev** [vercel/community#2419] [Help]
### Problem Description

I need to update my project to next 13, but every time i try to deploy to vercel I get this error 
`Error: The Serverless Function "api/login" is 54.01mb which exceeds the maximum size limit of 50mb. Learn More: https://vercel.link/serverless-function-size`
`NOW_SANDBOX_WORKER_MAX_LAMBDA_SIZE: The Serverless Function "api/login" is 54.01mb which exceeds the maximum size limit of 50mb.`

**Comment by amyegan** [comment]
Closing as duplicate. Thanks for sharing your solution in https://github.com/vercel/community/discussions/103#discussioncomment-5865349
